src/search/search-index-new/search/map-results-to-display.ts

```diff
@@ -0,0 +1,80 @@
+import db from '..'
+import { Page, FavIcon } from '../models'
+
+export interface SearchDisplayResult {
+    url: string
+    title: string
+    hasBookmark: boolean
+    displayTime: number
+    screenshot: string
+    favIcon: string
+    tags: string[]
+}
+
+// TODO: move somewhere
```

Comment:
If you look at the direct-linking-backend repo, I tend to have seperate types.ts files on different levels depending on the locality of its usage. Maybe something for a style guide?